Dockerfile.alpine

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7.3-alpine
+FROM ruby:3.3-alpine
```

Comment:
I believe CI is attempting to pass the RUBY_VERSION build arg, but here it's always ruby 3.3.

Either remove the RUBY_VERSION arg from CI and script/cibuild-docker or support it here in Dockerfile.